spec/confirmation_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "confirmation"
+
+describe "Confirmation" do
+  describe ".initialize" do
+    it "should create a confirmation text detailing when the cusotmer should receive their food by" do
+      order_time = Time.new(2021, 10, 01, 12, 27)
+      # order_time = Time.now
+      puts order_time
+      confirmation = Confirmation.new(order_time)
+      expect(confirmation.content).to eq("Thank you! Your order was placed and will be delivered before 13:27")
```

Comment:
No other class is accessing this `.content` reader, so you shouldn't be testing it. You want to be testing _behaviour_ not _state_. This means that you only want to be testing the inputs and outputs of the interface to this class.